### PR TITLE
Fix kind of a typo

### PR DESCRIPTION
**Pull request by antaljanosbenjamin**
### Description

Please briefly explain the changes you made.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
(paste the link to the issue it closes)

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

**Comment by katarinasupe** [comment]
@antaljanosbenjamin, nice to see your typo fix here 😉 ❤️  Thank you! 🙏

**Review by kgolubic — APPROVED** [review]
Everything looks OK. Thank you, @antaljanosbenjamin, for reporting and fixing this.

File: pages/clustering/high-availability.mdx [hunk 1]
```diff
@@ -157,7 +157,7 @@ the flag to more than `--instance-down-timeout-sec`, as it is needed first to co
 
 Every `--instance-health-check-frequency-sec` seconds, the coordinator contacts each instance. 
 The instance is not considered to be down unless `--instance-down-timeout-sec` has passed and the instance hasn't responded to the coordinator in the meantime. Expectation currently is to set `--instance-health-check-frequency-sec`
-to be less than `--instance-down-timeout-sec` and for `--instance-down-timeout-sec to be multiplier` of `--instance-health-check-frequency-sec` with coefficiient N.
+to be less than `--instance-down-timeout-sec` and for `--instance-down-timeout-sec` to be multiplier of `--instance-health-check-frequency-sec` with coefficiient N.
```
Comment (katarinasupe) [inline pages/clustering/high-availability.mdx:160]:
```suggestion
to be less than `--instance-down-timeout-sec` and for `--instance-down-timeout-sec` to be multiplier of `--instance-health-check-frequency-sec` with coefficient N.
```

**Review by katarinasupe — CHANGES_REQUESTED** [review]
Since we're already fixing it 😂

**Comment by kostasrim** [comment]
Is this a crossover episode or you just stacking contributions @antaljanosbenjamin :rofl:

**Comment by kgolubic** [comment]
> Is this a crossover episode or you just stacking contributions @antaljanosbenjamin 🤣

Contrib accepted :)

**Pull request merged**